rocketpy/AeroSurfaces.py

```diff
@@ -1918,3 +1916,55 @@ def allInfo(self):
         self.aerodynamicInfo()
 
         return None
+
+
+class RailButtons:
+    """Class that defines a generic rail button.
+
+    Attributes
+    ----------
+    RailButtons.upper_button_position : int, float
+        Position of the upper rail button in meters. The upper button is the one
+        closest to the nose cone and furthest from the tail. The coordinate system
+        used is the same as the Rocket that the buttons will be a part of.
+    RailButtons.lower_button_position : int, float
+        Position of the lower rail button in meters. The lower button is the one
+        closest to the nose cone and furthest from the tail. The coordinate system
+        used is the same as the Rocket that the buttons will be a part of.
+    RailButtons.angular_position : int, float
+        Angular position of the rail buttons in degrees measured
+        as the rotation around the symmetry axis of the rocket
+        relative to one of the other principal axis.
+    """
+
+    def __init__(self, upper_button_position, lower_button_position, angular_position):
+        """Initializes RailButtons Class.
+
+        Parameters
+        ----------
+        upper_button_position : int, float
+            Position of the upper rail button in meters. The upper button is the one
+            closest to the nose cone and furthest from the tail. The coordinate system
+            used is the same as the Rocket that the buttons will be a part of.
```

Comment:
Using "the coordinate system used is the same as the Rocket..." is dangerous. As a good practice, we want all components that can be added to a Rocket to be rocket independent, which means it cannot make any assumptions regarding the rocket's coordinate system.

By rocket independent, I mean, we should be able to use any component, in this case, a rail button pair, in any rocket, regardless of the coordinate system, with it working flawlessly.

I believe the current implementation does not satisfy this pillar, which was followed rigorously during the development of the new motor classes.

Reply:
I agree, referencing the Rocket coordinate system will likely lead to the user going back through the Rocket coordinate system documentation only to discover that this position is as simple as the distance to the nozzle (or nosetip) in most cases.

Reply:
I don't know if this is a valid issue since rail buttons are a bit different from motors and aerosurfaces in the sense that they are not added to a rocket, but set to one. There is no method that adds a rail button based on a RailButton object, like there is with `addSurfaces` and `addMotor`.

What could be done is add geometrical params to the `RailButtons` class init, but they would not be used in any calculations. They would be eventually useful whenever drag calculations are added, so implementing this would be future-proofing some break changes (but maybe adding drag might cause breaking changes anyway)